gradle/minecraft.gradle

```diff
@@ -9,6 +9,18 @@ evaluationDependsOn api.path
 // Apply shared Gradle configuration in SpongeAPI
 apply from: api.file('gradle/sponge.gradle')
 
+repositories {
+    mavenLocal()
+}
+
+configurations.all {
+    resolutionStrategy {
+        force 'org.spongepowered:configurate-hocon:3.7-SNAPSHOT'
+        force 'org.spongepowered:configurate-gson:3.7-SNAPSHOT'
+        force 'org.spongepowered:configurate-yaml:3.7-SNAPSHOT'
```

Comment:
I don't like this being in `Minecraft.gradle` because it's not really related to being in Minecraft.gradle, but I see why it's being applied. That being said, I'd just override the dependency in SpongeCommon explicitly and exclude the dependency inherited from spongeAPI.

Reply:
This is an old attempt at trying to make the dependency version override -- but nothing I've tried has worked unfortunately. I'll poke at it some more -- I'm sure it's just a matter of fighting gradle + fg until they behave

Reply:
I'll try to give a better solution, just might need to wait till I can spend some time on it.

Reply:
I might be missing something totally obvious here, but why aren't you just bumping the dep in the API layer? Given I'd like 7.3 to include Configurate 3.7, if this ends up merged in I want it available in our API 7.3 snapshots anyway.

Reply:
I thought part of the plan was to have 3.7 available for 7.2 builds as well? but if there aren't more of those, then sure, we can just have the dep in-api and i'll strip out all the buildscript changes

Reply:
The 7.2 builds will include the 7.3 snapshot, it's just we guarantee the 7.2 portion of the API. So yeah, just stick the dep in the API.